Foundation/FileHandle.swift

```diff
@@ -8,13 +8,26 @@
 //
 
 import CoreFoundation
+#if os(Windows)
+import WinSDK
+#endif
```

Comment:
Should this be added to `Foundation/NSSwiftRuntime.swift` instead to keep all of the imports in one place?

Reply:
Sure, that is fine by me.  I just don't want this exported is all (I don't think that Foundation should pull in the entire Windows SDK).  Alas, the private imports will have to wait :-(.

Reply:
Ah yes, just noticed it was a different module to the `MSVCRT` one!